module/x/gravity/keeper/cosmos_originated.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/cosmos/gravity-bridge/module/x/gravity/types"
 )
 
-func (k Keeper) GetCosmosOriginatedDenom(ctx sdk.Context, tokenContract string) (string, bool) {
+func (k Keeper) getCosmosOriginatedDenom(ctx sdk.Context, tokenContract string) (string, bool) {
```

Comment:
I think this is a function that is useful to keep public

Reply:
Agree this could be helpful to consumer modules.